website/docs/r/oci_auth_backend_config.html.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: "vault"
+page_title: "Vault: vault_oci_auth_backend_config resource"
+sidebar_current: "docs-vault-resource-oci-auth-backend-config"
+description: |-
+  Configures the OCI (Oracle Cloud Infrastructure) Auth Backend in Vault.
+---
+
+# vault\_oci\_auth\_backend\_config
+
+Configures the OCI (Oracle Cloud Infrastructure) Auth Backend in Vault.
+
+This resource sets the home tenancy id that Vault will use when making
+API requests on behalf of an OCI Auth Backend.
+
+For more information, see the
+[Vault docs](https://developer.hashicorp.com/vault/api-docs/auth/oci#configure-home-tenancy-method).
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+## Example Usage
+
+```hcl
+resource "vault_auth_backend" "example" {
+  type = "oci"
+}
+
+resource "vault_oci_auth_backend_config" "example" {
+  backend         = vault_auth_backend.example.path
+  home_tenancy_id = "ocid1.tenancy.oc1..aaaaaaaah7zkvaffv26pzyauoe2zbnionqvhvsexamplee557wakiofi4ysgqq"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `description` - (Optional) A description of the auth backend.
+
+* `namespace` - (Optional) The namespace to provision the resource in.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+   *Available only for Vault Enterprise*.
+
+* `home_tenancy_id` - (Required) The Tenancy OCID of your OCI account.
+
```

Comment:
The `tune` fields should be documented here as well. See [gcp](https://registry.terraform.io/providers/hashicorp/vault/latest/docs/resources/gcp_auth_backend#tune-5) for reference and feel free to copy it over from the same page, including the block description and then all inner fields.